google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/SchemaAwareStreamWriter.java

```diff
@@ -124,19 +126,19 @@ private void refreshWriter(TableSchema updatedSchema)
     this.streamWriter = streamWriterBuilder.setWriterSchema(this.protoSchema).build();
   }
 
-  private Message buildMessage(T item)
+  private List<DynamicMessage> buildMessage(Iterable<T> items)
       throws InterruptedException, DescriptorValidationException, IOException {
     try {
       return this.toProtoConverter.convertToProtoMessage(
-          this.descriptor, this.tableSchema, item, ignoreUnknownFields);
-    } catch (Exceptions.DataHasUnknownFieldException ex) {
+          this.descriptor, this.tableSchema, items, ignoreUnknownFields);
+    } catch (RowIndexToErrorException ex) {
```

Comment:
How do we know we have DataHasUnknownFieldException?

Reply:
Accidentally removed, added back